Cargo.toml

```diff
@@ -2,6 +2,7 @@
 name = "objc"
 version = "0.1.6"
 authors = ["Steven Sheldon"]
+build = "build.rs"
```

Comment:
My preference would be to split the NSObject functionality into another crate that's optional and used with the gnustep feature. That crate can have its own build script then. That way, the objc crate doesn't need a build script, which is nice because I'm not sure how cargo knows when to rebuild libraries with a build script. (Does it try to rebuild them every time since it can't tell if the build.rs will change anything?)

If you can just extract this into a crate in a subdirectory, included in a path, I can take care of publishing it later if we need to.